### ISSUE TITLE
I get a "failed print" notification when my printer powers off 30 minutes after a print is completed. 

### ISSUE THREAD

**Issue by simpat1zq** [Clon1998/mobileraker_companion#46]
My printer is on a smart plug, and when the print is complete, it gets automatically turned off after 30 minutes. However, mobileraker sends a notification saying the previous print failed. Is there any way to disable that?

**Comment by Clon1998** [comment]
Hey,
thanks for reaching out.
I think the major rewrite I am working on in #45 will resolve this issue.

Hence, if you don't mind you can switch to the `rewrite` branch and use it for a while and give me some feedback on how it is going and if you encounter any other problems.

To get the rewrite branch to work please run:
```bash
cd ~/mobileraker_companion
git pull 
git switch rewrite
./scripts/install-mobileraker-companion.sh
```

**Comment by simpat1zq** [comment]
Thanks. When I do the git switch rewrite command, I get an error

fatal: invalid reference: rewrite

**Comment by Clon1998** [comment]
I have merged #45.
Therefore, the branch `rewrite` does not exist anymore.

Just switch to main and do a git pull:
```
git fetch
git switch main
git pull
sudo systemctl restart mobileraker.service
```
